Code/Tools/AzTestRunner/src/main.cpp

```diff
@@ -129,6 +129,13 @@ namespace AzTestRunner
             std::cout << "LIB: " << lib << std::endl;
         }
 
+        // Construct a retry command if test fails
+        std::string retry_command = "Retry command: " + std::string(argv[0]) + " " + lib + " " + symbol;
+        for (int i = 1; i < argc; i++)
```

Comment:
I hate this variable name but I'm not going to expect you to rewrite it as part of this review.